typings/ipfsd-ctl/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'ipfsd-ctl';
```

Comment:
This package has built-in type declarations, why do we need to declare module like this? cc: `ipfs`, `ipfs-http-client`

Reply:
It throws error in the implementation even after successful build and existing in the node_modules. The same case is there in `ew-did-regsitry` as well. https://github.com/energywebfoundation/ew-did-registry/tree/development/typings

Reply:
A little bit weird, maybe we can add a task to jira to fix that or at least try to fix

Reply:
I took some time and tried to fix it, we can have a dedicated task for it maybe later based on the priorities.